server/controllers/finance/reports/vouchers/index.js

```diff
@@ -126,3 +135,18 @@ function report(req, res, next) {
     .done();
 }
 
+function creditNotedRef(uuid) {
+  const sql = `
+    SELECT BUID(v.uuid) as uuid, v.date, v.project_id, v.currency_id, v.amount,
+      v.description, v.user_id, v.type_id,  u.display_name, transaction_type.text,
+      CONCAT_WS('.', '${entityIdentifier}', p.abbr, v.reference) AS reference, reversed
+    FROM voucher v
+    JOIN project p ON p.id = v.project_id
+    JOIN user u ON u.id = v.user_id
+    LEFT JOIN transaction_type ON v.type_id = transaction_type.id
+    WHERE v.reference_uuid = ?;
+`;
+  return db.one(sql, db.bid(uuid)).then(voucher => {
```

Comment:
The `.then()` here doesn't do anything.  In general, if you have a function that simply returns it's input, it probably can be removed, unless it is needed for a hack.